config/initializers/markdown.rb

```diff
@@ -2,8 +2,13 @@
 require 'rouge'
 
 class Markdown
+  LINK_REGEXP = %r((?<![=["|']|><a-z0-9_\.])(?<!http:\/\/)((http[s]?\:\/\/)?([a-z0-9\.]+\.[a-z]{2,5}(\/[\?=&a-z0-9_\.\/]+)?)))
```

Comment:
I'm not sure that regexp is the best solution for this. Did you check Kramdown for working for links?

If it can't then okay, regexp will good for us 👍